js/client-chat.js

```diff
@@ -958,6 +958,13 @@
 					});
 					break;
 
+				case 'remove':
+					if (app.user.get('isStaff')) break;
```

Comment:
Does `isStaff` exist client-side? Probably not.

Anyway, this should be a preference. We probably shouldn't force censorship on users.